dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/JSONUtils.java

```diff
@@ -82,6 +82,12 @@ private JSONUtils() {
         throw new UnsupportedOperationException("Construct JSONUtils");
     }
 
+    public static synchronized boolean setTimeZone(TimeZone timeZone) {
+        objectMapper.setTimeZone(timeZone);
+        logger.info("set timezone: {}", timeZone);
```

Comment:
remove if not use.

Reply:
> remove if not use.

Do not understand this sentence, please elaborate